enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Filter/Filter.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { InteractiveQuestion } from "embedding-sdk";
+import { CommonSdkStoryWrapper } from "embedding-sdk/test/CommonSdkStoryWrapper";
+import { Box, Flex } from "metabase/ui";
+
+import { Filter } from "./Filter";
+
+const QUESTION_ID = (window as any).QUESTION_ID || 12;
+
+export default {
+  title: "EmbeddingSDK/InteractiveQuestion/Filter",
```

Comment:
Just to confirm, would this be named `FilterDropdown` or `Filter`?

Reply:
that should be sorted now 😄